### ISSUE TITLE
Valibot import declared in ModelToValibot uses default import instead of namespace 

### ISSUE THREAD

**Issue by UpstartMPotnick** [sinclairzx81/typebox-codegen#21]
The initial line generated for the buffer [here](https://github.com/sinclairzx81/typebox-codegen/blob/ab33260a00e306c63e6e1cf41b48c41a9489106f/src/model/model-to-valibot.ts#L229) imports valibot as a default export, i.e.
```ts
import v from 'valibot'
```

The docs recommend using a namespace import (or, more efficiently, named imports), i.e.:
```ts
import * as v from 'valibot'
// or
import { array, string, etc... } from 'valibot'
```

Oddly enough, VS Code doesn't show any errors on the generated schema using the current approach. However, `tsc` doesn't compile, throwing the following error:
```zsh
src/lib/schema.generated.ts:5:8 - error TS1192: Module '"/Users/.../node_modules/valibot/dist/index"' has no default export.

5 import v from 'valibot'
```

**Comment by sinclairzx81** [comment]
@UpstartMPotnick Hi, thanks for reporting.

Will take a look at this over the weekend. I think Valibot supports default ESM exports, but can update to barrel imports if this is more typical. I could also import individual types (as this seems to be the promoted feature of Valibot)

Will report back on this thread once I've taken a look.
Cheers
S

**Comment by sinclairzx81** [comment]
@UpstartMPotnick Hiya, this should be resolved on `@sinclair/typebox-codegen@0.8.7`.

Updates 
- Upgrade Zod 3.22.2 
- Upgrade Valibot 0.13.1
- Upgrade TypeBox 0.31.8

Additional

- Implement new types available on Valibot 0.13.1
- Use barrel import on Valibot

```typescript
import * as v from 'valibot'
```

Notes

Had a look at individual imports, but the generated output is somewhat difficult to visually parse when running on the Workbench. Have opted for the barrel `*` import over default import as per suggestion.

Can test generated output on the Workbench here https://sinclairzx81.github.io/typebox-workbench/

**Comment by sinclairzx81** [comment]
@UpstartMPotnick Heya, will close off this issue for now, but if you spot any problems with the updates, feel free to reply on this thread.

Cheers
S
